tlsutil/config_test.go

```diff
@@ -10,6 +10,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/hashicorp/yamux"
```

Comment:
This import isn't formatted right - the two lines should be grouped

Reply:
You are right, it is fixed now!